Assets/MixedRealityToolkit.SDK/Experimental/HandCoach/Scripts/MoveToTarget.cs

```diff
@@ -0,0 +1,197 @@
+
+        /// <summary>
+        /// Target to move to.
+        /// </summary>
+        public GameObject TargetObject
+        {
+            get
+            {
+                return targetObject;
+            }
+            protected set
+            {
+                targetObject = value;
+            }
+        }
+
+
+        [Tooltip("Shared parent between tracking and target objects used for relative local positions.")]
+        [SerializeField]
+        private GameObject rootObject = null;
+
+        /// <summary>
+        /// Shared parent between tracking and target objects used for relative local positions.
+        /// </summary>
+        public GameObject RootObject
+        {
+            get
+            {
+                return rootObject;
+            }
+            protected set
+            {
+                rootObject = value;
+            }
+        }
+
+        [Tooltip("Duration of move from tracking object to target object in seconds.")]
+        [SerializeField]
+        private float duration = 1.38f;
+
+        /// <summary>
+        /// Duration of move from tracking object to target object in seconds.
+        /// </summary>
+        public float Duration
+        {
+            get
+            {
+                return duration;
+            }
+            protected set
+            {
+                duration = value;
+            }
+        }
+
+        [Tooltip("Tunable offset to get the GameObject to arrive at the right target position.")]
+        [SerializeField]
+        private Vector3 targetOffset = new Vector3(0f, 0f, 0f);
+
+        /// <summary>
+        /// Tunable offset to get the GameObject to arrive at the right target position.
+        /// </summary>
+        public Vector3 TargetOffset
+        {
+            get
+            {
+                return targetOffset;
+            }
+            protected set
+            {
+                targetOffset = value;
+            }
+        }
+
+        [Tooltip("Lerp curve.")]
+        [SerializeField]
+        private AnimationCurve animationCurve = AnimationCurve.Linear(0f, 0f, 1f, 1f);
+
+        /// <summary>
+        /// Lerp curve.
+        /// </summary>
+        public AnimationCurve AnimationCurve
+        {
+            get
+            {
+                return animationCurve;
+            }
+            protected set
+            {
+                animationCurve = value;
+            }
+        }
+
+        // The local position of this gameObject relative to the root object
+        private Vector3 relativePositionTrackingToRoot;
+
+        // The local position of targetObject relative to the root object
+        private Vector3 relativeTargetPositionToRoot;
+
+        // bool to determine when to stop the follow sequence
+        private bool followingTargetObject;
+
+        // Since this script can attach to an object with an animator, we need to update position in LateUpdate
+        private void LateUpdate()
```

Comment:
@thalbern for consistency should we also use /// comments on private methods?

Reply:
yes I would say so - if it's already commented there's no harm to show it in the api docs as well

Reply:
I'll keep this one out of the API docs since LateUpdate is a Unity MonoBehaviour API